README.md

```diff
@@ -50,27 +75,183 @@ module.exports = getConfig({
+
+## Examples
+
+There are 3 example projects in the [/examples](https://github.com/HenrikJoreteg/hjs-webpack/tree/master/examples) directory.
+
+## Config options
+
+The main export you get when you `require('hjs-webpack')` is simply a pre-configured `webpack.config.js`. You could take the result of that and add other plugins if you so chose, but shouldn't be necessary for most common tasks.
+
+### `in`
+
+This should just be the path to the file that serves as the main entry point of your application.
+
+### `out`
+
+Path to directory where we're going to put generated files.
+
+### `isDev`
+
+A boolean to indicate whether or not everything is in production mode (minified, etc.) or development mode (everything hotloaded and unminified).
+
+### `html` (optional, can be boolean or function)
+
+This option is `true` by default. This means, by default, we'll serve and generate a very basic HTML file that looks like this:
+
+```html
+<!doctype html>
+<meta charset="utf-8"/>
+<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
+<link rel="stylesheet" href="/assets-and-index-html.1.0.0.css"/>
+<body></body>
+<script src="/app.js"></script>
+```
+
+Note the `<meta charset>` tag and mobile viewport settings are there by default.
+
+The empty `<body>` followed by the main script tag is also intentional. The ordering ensures we don't have to wait for `DOMReady` in our clientside code, you can safely assume that *both* `document.body` and `document.head` will be available when your script executes.
 
-This package includes an executable `hjs-dev-server`. If you've installed `hjs-webpack` in your app and if you've set up your config as above, you can run a dev server with the included executable by setting up the `scripts` section of your `package.json` something like:
+If you just want to do JS and CSS and handle all the html yourself, simply add `html: false` to your config (see examples directory for example).
 
-```json
-{
-  "name": "your-awesome-app",
-  "scripts": {
-    "start": "hjs-dev-server",
-    "build": "NODE_ENV=production webpack",
-    "deploy": "npm run build && surge -p public"
-  },
-  "dependencies": {
-    "webpack": "*",
-    "hjs-webpack": "*",
-    "surge": "*"
+**using an `html` function to generate specific files**
+
+This is where it gets interesting. Imagine pre-rendering all known structural content for a Native Web App to static files. Users get pixels on the screen immediately, your JS takes over when downloaded. If you're using React, this "taking over" can be completely seemless and invisible to the user. It's also possible with this approach to write an app that works entirely without JS. See the [prerendered-html-files example](https://github.com/HenrikJoreteg/hjs-webpack/tree/master/examples/prerendered-html-files).
```

Comment:
s/seemless/seamless in "If you're using React, this "taking over" can be completely seemless and invisible to the user."